### PR TITLE
Fix typos, punctuation, style. Link to Network Information API.

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

File: src/content/en/progressive-web-apps/checklist.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ The following tests are verifiable with the Lighthouse tool.
 <table class="responsive">
   <tbody>
     <tr>
-      <th colspan=2><h3>Pages each have a URL</h3></th>
+      <th colspan=2><h3>Each pages has a URL</h3></th>
```
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: The title was grammatically correct before. The new one isn't. Either change it back or change the new one to 'Each page has'.

File: src/content/en/progressive-web-apps/checklist.md [hunk 2]
```diff
@@ -310,13 +310,13 @@ and [social discovery](/web/fundamentals/discovery-and-monetization/social-disco
...
           <li>
-            A news app you should have the <a href="/search/docs/data-types/articles">
+            A news app should have the <a href="/search/docs/data-types/articles">
             NewsArticle type markup</a> for Rich Cards and/or 
             <a href="https://www.ampproject.org/docs/reference/spec.html">
             AMP support</a>.
           </li>
           <li>
-            An ecommerce app should have the
+            An e-commerce app should have the
```
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: 'ecommerce' is the correct one.

File: src/content/en/progressive-web-apps/checklist.md [hunk 3]
```diff
@@ -715,7 +715,7 @@ Note: This check list only applies if notifications implemented
 <table class="responsive">
   <tbody>
     <tr>
-      <th colspan=2><h3>Ensure that the UI that encourages users to turn on Push Notifications isn't overly aggressive.</h3></th>
+      <th colspan=2><h3>Ensure that the UI that encourages users to turn on Push Notifications isn't overly aggressive</h3></th>
```
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: Can you shorten this title. Maybe something like 'Avoid aggressive permission requests for notifications'. I don't really care how it's phrased. It just should be shorter.

File: src/content/en/progressive-web-apps/checklist.md (hunk 2 above)
Comment (dandv) [inline src/content/en/progressive-web-apps/checklist.md]: Citation needed?

http://english.stackexchange.com/questions/174165/correct-spelling-and-or-hyphenation-for-electronic-commerce

https://www.merriam-webster.com/dictionary/ecommerce

File: src/content/en/progressive-web-apps/checklist.md (hunk 3 above)
Comment (dandv) [inline src/content/en/progressive-web-apps/checklist.md]: That's how I found it, but I'll shorten it. Thanks for pointing that out.

File: src/content/en/progressive-web-apps/checklist.md (hunk 2 above)
Comment (jpmedley) [inline src/content/en/progressive-web-apps/checklist.md]: Devsite styleguide https://developers.google.com/style/wordlist

Comment (dandv) [inline src/content/en/progressive-web-apps/checklist.md]: Thanks!

**Pull request merged**